sw/device/tests/crt_test.c

```diff
@@ -54,6 +58,91 @@ static void init_uart(void) {
   base_uart_stdout(&uart0);
 }
 
+static void test_crt_section_clear(void) {
+  // Maximum end index of target section.
+  const size_t len = 32;
```

Comment:
Nit: constants should use the casing `kConst`.

Reply:
Done